odoo/tools/pdf.py

```diff
@@ -2,6 +2,20 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 import io
 from PyPDF2 import PdfFileWriter, PdfFileReader
+from PyPDF2.generic import DictionaryObject, NameObject, ArrayObject, DecodedStreamObject, NumberObject, createStringObject, ByteStringObject
+from logging import getLogger
+from hashlib import md5
+from zlib import compress, decompress
+from datetime import datetime
+try:
+    from fontTools.ttLib import TTFont
+except ImportError:
+    TTFont = None
```

Comment:
nitpicking: imports are normally sorted alphabetically inside each group, in order to make them easier to read/scan. Really minor, but as we're reordering them, might as well do it well ;)

we can keep fonttools separate as it gets special treatment

```suggestion
import io

from datetime import datetime
from hashlib import md5
from logging import getLogger
from PyPDF2 import PdfFileWriter, PdfFileReader
from PyPDF2.generic import DictionaryObject, NameObject, ArrayObject, DecodedStreamObject, NumberObject, createStringObject, ByteStringObject
from zlib import compress, decompress

try:
    from fontTools.ttLib import TTFont
except ImportError:
    TTFont = None
```